lldb/source/Utility/LLDBAssert.cpp

```diff
@@ -7,6 +7,7 @@
 //===----------------------------------------------------------------------===//
 
 #include "lldb/Utility/LLDBAssert.h"
+#include "lldb/Core/Telemetry.h"
```

Comment:
Utility cannot depend on Core. If you want something like this, I guess you can put it somewhere under Debugger::AssertCallback